### PR TITLE
refactor(frontend): Include native token in token group property

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

It is useful to have the native token of the token group as defined prop, even if theoretically it is always the first element of the list, just to have a place to get it in an "obvious" way.

Note: in the same PR, we adjust the breakdown of the tests for function `groupTokensByTwin`, related to this change.

### CODE REVIEW

File: src/frontend/src/lib/types/token.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export type OptionTokenUi = Option<TokenUi>;
 
 export interface TokenGroupUi {
 	header: TokenMetadata;
+	nativeToken: TokenUi;
```
Comment (peterpeterparker) [inline src/frontend/src/lib/types/token.ts:74]: We don't need `nativeNetwork` anymore then? We add a TODO to remove it?

Comment (peterpeterparker) [reply src/frontend/src/lib/types/token.ts:74]: Likewise, do we still need `header` afterwards? TODO to be removed?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/types/token.ts:74]: i would remove it afterwards, but YES we don't need them anymore. Adding a TODO

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Comment by AntonioVentilii-DFINITY** [comment]
[OISY-69]

**Pull request merged**